### PR TITLE
Branded splash screen

### PR DESCRIPTION
**Pull request by alenmagdic**
**Description**
Added aconno logo to splash screen

**Issues to solve**
#371

### CODE REVIEW

**Comment by borna-mdn** [comment]
Can you screenshot me the splash screen?

File: app/src/main/res/layout/activity_splash.xml [hunk 1]
```diff
@@ -5,28 +5,31 @@
     android:layout_height="match_parent"
     android:theme="@style/SplashTheme">
 
-    <androidx.constraintlayout.widget.Guideline
-        android:id="@+id/start"
+    <LinearLayout
```
Comment (borna-mdn) [inline app/src/main/res/layout/activity_splash.xml:8]: Nesting this inside `ConstraintLayout` is unnecessary.

Comment (alenmagdic) [reply app/src/main/res/layout/activity_splash.xml]: Ok, I have removed ConstraintLayout and set LinearLayout as root layout.

**Comment by alenmagdic** [comment]
> Can you screenshot me the splash screen?

Screenshot_20200720-125059_Sensorics

**Pull request merged**